tasks/account.js

```diff
@@ -228,6 +228,62 @@ ACCOUNT_SCOPE.task("bot:remove", "Remove bot from the CMAccount")
         }
     });
 
+ACCOUNT_SCOPE.task("payment-token:add", "Add payment token to CMAccount")
+    .addOptionalParam("privateKey", "Private key to use", process.env.CMACCOUNT_PK)
+    .addOptionalParam("cmAccount", "CMAccount address", process.env.CMACCOUNT_ADDRESS)
+    .addParam("paymentToken", "Payment token address")
+    .setAction(async (taskArgs, hre) => {
+        const cmAccount = await getCMAccount(taskArgs.cmAccount);
```

Comment:
What happens if cmAccount is not given?
Also what happens if params given pk and address are not for the same account?

Reply:
It just fails. Same for the private key. 

It's just more convenient when you set `CMACCOUNT_PK` and `CMACCOUNT_ADDRESS` in your envs (export CMACCOUNT_ADDRESS=...`

Reply:
> Also what happens if params given pk and address are not for the same account?

Transaction fails

Reply:
Shouldn't we add a check and return an error describing the issue? Or connect(signer) returns a proper error?